app/profile/manageTask/page.tsx

```diff
@@ -40,7 +41,18 @@ export default function ManageTask() {
   const [currentImportance, setCurrentImportance] = useState("all");
   const [currentTasksData, setCurrentTasksData] = useState<DataType[]>([]);
   const userId = useSelector(getUserId);
+  const doneTasks = useSelector(getUserDoneTasks);
 
+  const updateUserDoneTasksValue = async () => {
+    try {
+      const response = axios.put("/api/users/login", {
+        userId: userId,
+        doneTasks: doneTasks,
+      });
+    } catch (error) {
+      console.log("something went wrong, we dont update your done tasks");
+    }
+  };

```

Comment:
To bym również przeniósł gdzieś na zewnątrz, do jakiegoś osobnego pliku i funkcji. Wywołanie tej akcji przez Reduxa tez może być spoko, bo wtedy wyłapujesz loading state (pending, fullfiled itd) i możesz np zrobić tak, że jak jest fullfilled to od razu w Redux nadpisujesz dane tego usera/taska majac natychmiastowy update na ekranie, zamiast robic kolejny refetch do bazy:)